app/src/test/java/org/pesmypetcare/mypetcare/features/pets/TestPetUpdate.java

```diff
@@ -0,0 +1,103 @@
+package org.pesmypetcare.mypetcare.features.pets;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.mypetcare.controllers.TrUpdatePet;
+import org.pesmypetcare.mypetcare.features.users.User;
+import org.pesmypetcare.mypetcare.services.StubPetManagerService;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class TestPetUpdate {
+    private Pet pet;
+    private User usr;
+    private TrUpdatePet trUpdatePet;
+
+    @Before
+    public void setUp() throws PetRepeatException {
+        usr = new User("johnDoe", "johndoe@gmail.com", "1234");
+        pet = new Pet();
+        pet.setName("Manolo");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "Manolo" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5219990)